src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/judgeapproval/RefusedOrderProcessor.java

```diff
@@ -117,10 +115,21 @@ public void processRefusedOrders(FinremCaseDetails finremCaseDetails, DraftOrder
             removedPsaItems.stream().filter(a -> a.getValue() != null).map(toRefusedOrderCollection).filter(Objects::nonNull)
         ).toList();
 
+        List<UuidCollection> newRefusalOrderIds = newRefusedOrders.stream()
+            .map(RefusedOrderCollection::getId)
+            .map(UuidCollection::new)
+            .toList();
+        List<UuidCollection> existingRefusalOrderIds = Optional.ofNullable(draftOrdersWrapper.getRefusalOrderIdsToBeSent())
```

Comment:
static imported java.util.Optional.ofNullable already. I think we can skip `Optioanl.`?